### DISCUSSION TITLE
Is Compiled still in active development

### DISCUSSION THREAD

**Discussion by J-P-P-V** [atlassian-labs/compiled#774] [General]
We are about to pick the tech stack for a big project at work, and compiled seems like it would be the best option around over all, but from a cursory glance, it would also seem like many issues are currently still open.

This is of course to be expected for a library at its early stage like Compiled, and would also be okay for our project, as it itself would take time to mature, but my worry is that the development of the library seems somewhat stale at the moment with mostly automated maintenance commits, and with key issues and pull requests such as support for extraction in next.js (#669) or support for css chunks that might fix it (#724) untouched for a long time.

This is of course not a complaint and I totally appreciate all the work put into this project. I do realize Compiled is still in its early (alpha?) stages, but just try to make an assessment of the situation.

**Comment by itsdouges** [comment]
Hey mate! Thanks for raising the discussion. It's not currently in active development at Atlassian as the developers working on it have been pulled into other projects for now.

Later in the year we're aiming to re-kick start it under a new owning team, so watch this space.

**Reply by brianbento** [reply]
Please sooner rather than later. This library looks awesome!

**Comment by TxHawks** [comment]
I see that work on compiled has resumed. Is improving Next.Js support in the cards soon?

**Reply by MonicaOlejniczak** [reply]
Compiled is in active development again, and is being invested in to improve the performance of Atlassian's products.

We plan on revisiting CSS extraction in the near future. The team will keep Next.js support in mind when we tackle this area, to make sure we are not missing anything in addition to our own complex requirements.
